W-16428790 mule sdk maven (#254)
mulesoft/docs-mule-sdk: modules/ROOT/pages/getting-started.adoc

NOTE: The maximum supported version for running your applications is JDK 17 for Mule runtime 4.6.x and later. During the packaging of your app, all code, including third-party dependencies, *must* be compiled for Java language level 1.8.

* Apache Maven 3.8.7 or later to manage your project’s build.
+

> NOTE: The mule-extensions-parent is not compatible with Apache Maven 3.9.x.

Reviewer: ```suggestion
NOTE: The `mule-extensions-parent` isn't compatible with Apache Maven 3.9.x.
```
